### ISSUE TITLE
Publish a real version?

### ISSUE THREAD

**Issue by fabricedesre** [mozilla/cargo-vet#148]
The book installation section recommends to use `cargo install cargo-vet`, but that builds the skeleton "Hello World!" Rust crate:

```console
/Users/fabrice > cargo install cargo-vet
    Updating crates.io index
  Downloaded cargo-vet v0.1.0
  Downloaded 1 crate (1.5 KB) in 0.47s
  Installing cargo-vet v0.1.0
   Compiling cargo-vet v0.1.0
    Finished release [optimized] target(s) in 4.47s
  Installing /Users/fabrice/.cargo/bin/cargo-vet
   Installed package `cargo-vet v0.1.0` (executable `cargo-vet`)
/Users/fabrice > cargo vet
Hello, world!
```

**Comment by Gankra** [comment]
@bholley only you have permission to do this

**Comment by bholley** [comment]
We published 0.2.0 a little while ago. I'm going to publish 0.3.0 later today.
